velox/dwio/common/tests/TestBufferedInput.cpp

```diff
@@ -31,3 +35,67 @@ TEST(TestBufferedInput, ZeroLengthStream) {
   EXPECT_FALSE(ret->Next(&buf, &size));
   EXPECT_EQ(size, 0);
 }
+
+TEST(TestBufferedInput, ParallelLoad) {
+  const int32_t kOneKB = 1 * 1024;
+  const int32_t threads = 4;
+  FLAGS_load_parallel = true;
```

Comment:
The name of this gflag is very generic. It is hard to tell what this is for. Can we avoid using a gflag and pass this bool as a parameter? If not, would it be possible to come up with a more descriptive name and perhaps move it to velox/flag_definitions/flags.cpp?